processor/src/giteaDB.ts

```diff
@@ -85,6 +93,18 @@ export async function subscribeToRepoDeletions(
   return sql.subscribe('delete:repository', callback)
 }
 
+/**
+ * Subscribe to all events on the repository table.
+ * the callback will be called.
+ * @param callback
+ * @returns
+ */
+export async function subscribeToRepo(
+  callback: (row: Row | null, info: ReplicationEvent) => Promise<void>,
+): Promise<SubscriptionHandle> {
+  return sql.subscribe('*:repository', callback)
+}
```

Comment:
I think it'd be a bit nicer if this was the generic subscribe:

```ts
export async function subscribeToRepo(
  eventName, callback: (row: Row | null, info: ReplicationEvent) => Promise<void>,
): Promise<SubscriptionHandle> {
  return sql.subscribe('${eventName}:repository', callback)
}
```

So you can do 

```ts
giteaDB.subscribeToRepo('*', cb)

giteaDB.subsribeToRepo('delete', cb)
```

Bonus points if `eventName` is given a type that enumerates the options.

Reply:
Resolved in 9872c37a.